FidoCore.md

```diff
@@ -1384,3 +1475,154 @@ begin
   Slots.Register(FSlots, FViewModel, Self)
 end;
 ```
+
+Slots can be of two types:
+
+- `stSynched`, the slot will be synchronized with the main thread  
+- `stNotSynched`, the slot will not be synchronized with the main thread.
+
+## Events driven architecture
+
+Units folder: `EventsDriven`.
+
+![](C:\Users\mendrix\Documents\fidolib - github\diagrams\Events driven.svg)
+
+> An event-driven architecture uses events to trigger and communicate between decoupled services and is common in modern applications built with microservices. An event is a change in state, or an update, like an item being placed in a shopping cart on an e-commerce website. Events can either carry the state (the item purchased, its price, and a delivery address) or events can be identifiers (a notification that an order was shipped).
+>
+> Event-driven architectures have three key components: event producers, event routers, and event consumers. A producer publishes an event to the router, which filters and pushes the events to consumers. Producer services and consumer services are decoupled, which allows them to be scaled, updated, and deployed independently.  [Source](https://aws.amazon.com/event-driven-architecture/)
+
+
+
+With FidoLib you can design your system adhering to the Events driven architecture.
+
+Out of the box you can use:
+
+- a Redis implementation that would use Queues, PubSub or an hybrid Queues + PubSub system to manage the events
```

Comment:
Could you add a notice that you need to get redit for this to work?
I know it's obvious, but maybe point people to the github repo :)

Reply:
Do I? Man you don't trust your fellow humans...

Reply:
I don't trust my own stupidity, but its quite obvious indeed